app/views/layouts/_top_bar.html.erb

```diff
@@ -58,7 +58,7 @@
         </a>
 
         <a href="<%= sign_up_path(state: "new-user") %>" class="crayons-btn mr-2 whitespace-nowrap" data-no-instant>
-          Create account
+          <span class="button-text">Create account</span>
```

Comment:
I think the CSS solution is the right call, but maybe there is a more descriptive CSS class name we can use here! I think @lisasy or @ludwiczakpawel could probably help us find the right name for this (naming stuff is the hardest part of web dev).

Ideally, we can apply the CSS class to the `<a>` tag instead of adding the `<span>` element to the page (since an anchor typically wraps text directly).

Reply:
Yes, I think we can go with utility class here, like this:

```
<a href="..." class="crayons-btn mr-2 whitespace-nowrap fs-s s:fs-base" data-no-instant>Create account</a>
```

`fs-s s:fs-base` - this means we will serve `var(--fs-s)` (equals to `14px`) for smallest breakpoint we support (up to 640px) and after reaching that breakpoint we will go back to default `var(--fs-base)` (equals to `16px`).

With that you can drop this extra CSS as well as that span element.

Reply:
Okay! I'll do it.

Reply:
Thanks for the helpful advice!